common/utils.js

```diff
@@ -440,26 +440,35 @@ const getMultisigProof = async (config, chain, multisigSessionId) => {
 
 const calculateDomainSeparator = (chain, router, network) => keccak256(Buffer.from(`${chain}${router}${network}`));
 
-const getItsEdgeContract = (chainConfig) => {
-    const itsEdgeContract =
+const getItsChains = (chainConfig) => {
```

Comment:
```suggestion
const getItsEdgeContract = (chainConfig) => {
```
naming doesn't make sense, we aren't getting many its chains